src/components/Application/Application.jsx

```diff
@@ -1,32 +1,62 @@
-import React, { PropTypes } from 'react';
+import React, { PropTypes, Component } from 'react';
 import { connect } from 'react-redux';
 
+import { ipcRenderer } from 'electron';
+
 import OperatorPanel from '../OperatorPanel/OperatorPanel.jsx';
 import ClientsPanel from '../ClientsPanel/ClientsPanel.jsx';
 import MessagePanel from '../MessagePanel/MessagePanel.jsx';
-import Settings from '../Settings/Settings.jsx';
+import SettingsPanel from '../SettingsPanel/SettingsPanel.jsx';
+
+import { setConfig } from '../../store/Chat/actions.js';
+
 import './Application.css';
 
-const Application = (props) => {
-  const renderMainView = () => (
-    props.settingsOpen ? <Settings /> :
+class Application extends Component {
+  constructor (props) {
+    super(props);
+
+    // Setup our IPC listener
+    ipcRenderer.on('config', this.updateConfig());
+    ipcRenderer.send('init-config');
+  }
 
+  updateConfig () {
+    const { updateConfig } = this.props;
+
+    return function (event, config) {
+      // Send config to Redux state
+      updateConfig(config);
+    };
+  }
```

Comment:
yeah, hmmm. You could also possibly just attach it to the payload as a key on the object.